packages/SwingSet/test/test-state.js

```diff
@@ -497,13 +497,13 @@ test('kernelKeeper promises', async t => {
   t.deepEqual(k.getKernelPromise(p1).subscribers, ['v3', 'v5']);
 
   const expectedAcceptanceQueue = [];
-  const m1 = { method: 'm1', args: { body: '', slots: [] } };
+  const m1 = { methargs: { body: '["m1"]', slots: [] } };
```

Comment:
as discussed, we shouldn't omit the arguments array, as that'll be treated as an eventual-get instead of an eventual-send

I suspect there's new way to factor our little helper functions (`capargs`/`capargsOneslot`/etc) now that we're building methargs instead of args, but I doubt it's worth putting too much time into now. Maybe for a later cleanup pass.